Application/Frontend/components/Messaging/MessagingInterface.tsx

```diff
@@ -265,7 +265,7 @@ export function MessagingInterface({ sender, receiver, privateChat, onMessageExc
               <Group grow>
                 <Textarea
                     ref={inputBoxRef}
-                    placeholder="Message @user2"
+                    placeholder={`Message @${receiver}`}
```

Comment:
- Good change to implement the stateManagement feature instead of hard code "user2"